proto/archive.proto

```diff
@@ -0,0 +1,54 @@
+syntax = "proto3";
+
+import "proto/context.proto";
+
+package archive;
```

Comment:
wanna call this zip or are there other archive formats you think would make sense here?

Reply:
Alright, both of you commented on this now, so I updated it all.